Cyberpunk2k20_v2/Cyberpunk2k20.html

```diff
@@ -0,0 +1,3893 @@
+<div class="body">
+    <div>
+
+        <input type="hidden" name="attr_ref_penality" value="0">
+        <input type="hidden" name="attr_int_penality" value="0">
+        <input type="hidden" name="attr_tech_penality" value="0">
+        <input type="hidden" name="attr_cool_penality" value="0">
+        <input type="hidden" name="attr_body_penality" value="0">
+        <input type="hidden" name="attr_attr_penality" value="0">
+        <input type="hidden" name="attr_luck_penality" value="0">
+        <input type="hidden" name="attr_ma_penality" value="0">
+        <input type="hidden" name="attr_emp_penality" value="0">
+
+        <div style="display:inline-block;width:460px;margin-top:30px">
+            <h1 style="margin-left: 30px;display: block;font-size: 50px;height: 50px;">Cyberpunk 2020</h1>
+            <input type="checkbox" style="margin-top: -15px;" class="edit-view"/>
+            <!--
+            <img style="display:block;margin-left: 30px;" src="https://banner2.kisspng.com/20180921/hh/kisspng-cyberpunk-2-77-game-electronic-entertainment-expo-cyberpunk-2-77-eine-vision-des-jahres-2-77-gam-5ba4890e8747d2.7977563615375096465541.jpg"   />
+            -->
+            <h4>Class:</h4>
+            <span class="bio-title" class="bioView" name="attr_class"></span>
+            <input class="bioEdit" type="text" name="attr_class" style="width:150px;">
+            <h4>Genre:</h4>
+            <span class="bio-title" class="bioView" name="attr_sex"></span>
+            <select name="attr_sex" style="width:70px;">
```

Comment:
Greetings @micka9fr 

Per building Character Sheet standards any attr_ related to [gender/sex should to be Inputs](https://wiki.roll20.net/Building_Character_Sheets#1._Code_of_Conduct.) rather than Selects.

Reply:
Is it better to use input text for that ?

Reply:
Yes. Input type=text is the preferred option.

Reply:
Sorry for time I was in holyday. I have push the change select into input type=text